src/template-strings/info.js

```diff
@@ -11,7 +11,9 @@ const infoMsg = {
 
   ApiSetdefault: 'Default version of {{owner}}/{{name}} set to {{version}}',
 
-  ApiUpdate: 'Updated API {{owner}}/{{name}}/{{version}}',
+  ApiUpdate: 'Updated API {{owner}}/{{name}}/{{version}} and visibility is set to {{visibility}}',
+
+  visibilityUpdate: 'Updated API visibility of {{owner}}/{{name}}/{{version}} is set to {{visibility}}',
```

Comment:
This message doesn't read well for me, what about
```suggestion
  visibilityUpdate: 'Visibility of {{owner}}/{{name}}/{{version}} set to {{visibility}}',
```
or if you stick with your message, I think a comma would be good after API. So it would be
`'Updated API, visibility of {{owner}}/{{name}}/{{version}} is set to {{visibility}}'`
Any thoughts @ponelat @barryadk ?

Reply:
I don't think the output should change when the user is only specifying the updated definition. Either keep the original message 'API updated', or list the settings that have been changed:
```
> swaggerhub api:update Org/ABC/2.0.0 --file api.yaml --setdefault
API definition updated
Default version set to 2.0.0

Reply:
This message is for when only the visibility is updated, the other message is changed also
`ApiUpdate: 'Updated API {{owner}}/{{name}}/{{version}} and visibility is set to {{visibility}}',`